### DISCUSSION TITLE
rabbitmq_tracing runs into 'erofs' when tracing files are listed in a containerized environment

### DISCUSSION THREAD

**Discussion by razvanphp** [rabbitmq/rabbitmq-server#11546] [Other] [answered]
### Describe the bug

when enabling rabbitmq_tracing plugin, one cannot access it in the admin GUI, we get this error:

```
Node 'rabbit@rabbitmq-0.(my-cluster-details).svc.cluster.local' could not be contacted: {badrpc,
{'EXIT',
{{badmatch,
{error,
erofs}},
[{rabbit_tracing_files,
list,
0,
[{file,
"rabbit_tracing_files.erl"},
{line,
18}]}]}}}
```

Checked and the plugin is enabled in CLI:

```
I have no name!@rabbitmq-1:/$ rabbitmqctl trace_on -p /
Starting tracing for vhost "/" ...
Trace enabled for vhost /
I have no name!@rabbitmq-1:/$ 
```

### Reproduction steps

1. helm install bitnami/rabbitmq
2. replicas set to 3
3. Wait for all pods to come online
4. Open rabbit UI through the web browser
5. Go to Admin
6. Click Tracing

### Expected behavior

Tracing GUI works/loads.

### Additional context

Originally reported here: https://github.com/bitnami/charts/issues/27082 but it does not seem to be a problem with the chart itself.

**Comment by michaelklishin** [comment, marked as answer]
`erofs` is the line you are looking for. The tracing plugin cannot perform a filesystem operation because the kernel refuses it with this error code, a "read-only file system."

The default path used by the tracing plugin is `/var/tmp/rabbitmq-tracing` and it can be changed using the `rabbitmq_tracing.directory` key via `advanced.config` (it is not exposed to `rabbitmq.conf`, for no reason in particular).

**Comment by michaelklishin** [comment]
@razvanphp GitHub does not send notifications for Discussion responses so I have to use this mention.
